src/df.jl

```diff
@@ -144,7 +144,7 @@ export fix_date_column
 
 function fix_age_column(df)
     levels = ["infant", "adolescent", "adult"]
-    ages = categorical(df[!, :age]; ordered=true, levels)
+    ages = categorical(df[!, :age]; levels, ordered=true)
```

Comment:
Was this automatically changed by your IDE too?

Reply:
Nope I prefer this way because the docstring and function signature for `categorical` is:

```julia
help?> categorical
search: categorical CategoricalPool CategoricalValue CategoricalArray CategoricalVector CategoricalMatrix

  categorical(A::AbstractArray; levels=nothing, ordered=false, compress=false)
```